src/client/styles/components/SubjectHeadings.scss

```diff
@@ -95,6 +95,7 @@ $table-row-border: 1px solid rgb(215, 210, 205);
   margin-bottom: 3px;
   line-height: 5px;
   padding: 4.5%;
+  padding-left: 8%;
```

Comment:
I think vertical ellipse is supposed to be aligned with tip of arrow. This property throws it way to the right of that. I want to avoid using padding and margin as much as possible until designs are finalized. It can get tricky to apply tweaks on top of tweaks on top of tweaks

Reply:
Changed the styling to align the vertical ellipse with the tip of the arrow, but I wasn't able to eliminate the padding.